daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1868,6 +1868,61 @@ char     *QDECL PRINTF_LIKE(1) va( const char *format, ... )
 =====================================================================
 */
 
+std::string InfoMapToString( const InfoMap& map )
+{
+	std::string info_string;
+
+	for ( const auto& pair : map )
+	{
+		if ( pair.first.find( INFO_SEPARATOR ) != std::string::npos ||
+			pair.second.find( INFO_SEPARATOR ) != std::string::npos )
+		{
+			Log::Notice( "Can't use keys or values with a \\\n" );
```

Comment:
Log::Warn and a more explicit message? I think the Info_ code has stuff to add escaping so you might wnat to do the same.